src/charts/scatter-plot.js

```diff
@@ -489,6 +593,59 @@ define(function(require) {
             dispatcher.call('customClick', e, d, d3Selection.mouse(e), [chartWidth, chartHeight]);
         }
 
+        /**
+         * Applies glow to hovered data point
+         * @return {void}
+         * @private
+         */
+        function highlightDataPoint(data) {
+            removePointHighlight();
+
+            if (!highlightFilter) {
+                highlightFilter = createFilterContainer(svg.select('.metadata-group'));
+                highlightFilterId = createGlowWithMatrix(highlightFilter);
+            }
+
+            let highlightCircle = svg.select('.metadata-group')
+                .selectAll('circle.highlight-circle')
+                .data([data])
+                .enter()
+                  .append('circle')
+                    .attr('class', 'highlight-circle')
+                    .attr('stroke', ({name}) => nameColorMap[name])
+                    .attr('fill', ({name}) => nameColorMap[name])
+                    .attr('fill-opacity', circleOpacity)
+                    .attr('cx', ({x}) => xScale(x))
+                    .attr('cy', ({y}) => yScale(y))
+                    .attr('r', ({y}) => areaScale(y))
+                    .style('stroke-width', highlightStrokeWidth)
+                    .style('stroke-opacity', highlightCircleOpacity);
+
+            highlightCircle
+                .attr('filter', `url(#${highlightFilterId})`);
+
+            highlightLabelsForDataPoint(data);
+        }
+
+
+        /**
+         * Draws the perpendecular lines and labels
+         * for the highlighted data point
+         * @return {void}
+         * @private
+         */
+        function highlightLabelsForDataPoint() {
```

Comment:
You could wait until somebody asks for it. At the end of the day, with the tooltip there is already a way to see the values